examples/assets_dbt_python/.github/workflows/examples/branch_deployments.yml

```diff
@@ -0,0 +1,86 @@
+name: Serverless Branch Deployments
```

Comment:
@maximearmstrong to confirm, was it intentional to include example github workflows in this PR?

Reply:
Yes! Currently, we don't have an example of the ci/cd workflows for a Dagster & dbt project outside of Dagster University, so we wanted
- an example of what a repo would look like with all the required files
- an example that could be easily shared for support.